abci/example/kvstore/kvstore_test.go

```diff
@@ -229,101 +227,105 @@ func valsEqual(t *testing.T, vals1, vals2 []types.ValidatorUpdate) {
 	}
 }
 
-func makeSocketClientServer(app types.Application, name string) (abciclient.Client, service.Service, error) {
+func makeSocketClientServer(
```

Comment:
Very minor nit: context and T should probably precede other arguments. (Both want to be "first" but I don't really care about that relative order)